README.md

```diff
@@ -1,2 +1,91 @@
-# bender
-A Slack concourse CI resource that can trigger jobs
+# bender Resource
+
+A  Concourse resource that can trigger any job (deployments, releases, ...) using slack.
+
+## Source Configuration
+
+* `slack_token`: *Required.* The slack token to be used. For more info check [slack](https://api.slack.com/docs/token-types#bot)
+
+* `channel`: *Required.* The channel name to be used by the bot.
+
+* `bot_name`: *Optional.*, *default `bender`* The bot name will be used to identify and filter messages. All messages must be addressed to the bot `@bot_name some message`.
```

Comment:
*Optional.*, *default `bender`* doesn't read well. Maybe add commas and stops.

Reply:
All messages must be addressed to the bot `@bot_name some message`. doesn't read well. Maybe redo this sentence like that: All messages must be addressed to the bot, e.g.: `@bot_name some message`.